### DISCUSSION TITLE
Allow user to reset measurable habit entries

### DISCUSSION THREAD

**Discussion by amrsamii** [iSoron/uhabits#1557] [Feature Requests]
## Pre-submission checklist
- [x] I am submitting a bug report, not a feature request.
- [x] I have searched for similar issues, but did not find any matches.

## Description
If I entered a number for a measurable habit then I want to reset it, I wouldn't be able to do so. If I tried to put zero to reset it, it counts towards the days of habits instead of skipping it. The day in the calendar still has grey color, while it should not have a color as any other day that doesn't have data.

## Steps to reproduce
1. Create a measurable habit.
2. Enter a random number.
3. Enter zero to reset the habit.
4. The data is not reset.

## System information
 - Phone: [Redmi Note 8 Pro]
 - Phone Operating System: [Android 11]
 - App version: [e.g. 2.1.1]

## Screenshots
![IMG_20221012_170202](https://user-images.githubusercontent.com/31626818/195378895-bf9b38a0-450f-4bf5-b269-940587f0db24.jpg)
![IMG_20221012_170249](https://user-images.githubusercontent.com/31626818/195378899-9291d32e-850c-4478-bc75-3dd3268fa2ab.jpg)

**Comment by iSoron** [comment]
Thanks for the bug report, @amrsamii. I'm converting this into a feature request, since this behavior is expected to the developers; a value of zero is different from a missing value. But we could have a button to erase the value, like we currently do for yes/no habits.

**Comment by SimonDedman** [comment]
I don't know if this is a bug, but occasionally I'll accidentally enter a value, save, then change back to 0, save, and the widget will still be highlighted. However in trying to reproduce this now, it works fine. I'm not sure if this would be a memory issue...?

**Comment by MarkJeronimus** [comment]
I came here for this because I regularly accidentally enter a value for the wrong day, and there's no way or feasible workaround to undo it.

**Comment by Cancoillotte** [comment]
Perhaps we should wait, because it will be implemented in the end? Hope springs eternal...

**Comment by ghost** [comment]
This would be very useful, if this app is still getting updated.
